src/metabase/models/action.clj

```diff
@@ -34,11 +37,12 @@
 
 (mi/define-methods
  Action
- {:types      (constantly {:type :keyword
-                           :parameter_mappings :parameters-list
-                           :parameters :parameters-list
+ {:types      (constantly {:type                   :keyword
```

Comment:
don't we need to add entity-id properties for http/implicit/query action too?

Reply:
I've updated the PR so the subtype models are inlined, which means entity_id is not needed anymore.